cmd/integration_link.go

```diff
@@ -414,9 +416,33 @@ List of available integrations:
 
 			utils.CheckForConsent(c.Context, currentApp, utils.ConsentTypeContainers)
 
-			pullRequestID := c.Args().First()
+			pullRequestID, err := strconv.Atoi(c.Args().First())
+			if err != nil {
+				errorQuit(errgo.Notef(err, "invalid pull request id"))
```

Comment:
question: aren't there some cases where a merge request could contain something else than integer? For instance on GitLab? I'm not completely keen on checking here if it's an integer. Why don't we simply trust the backend which forwards the query to GitHub/GitLab to know if it's valid or not?

suggestion:

```suggestion
				errorQuit(errgo.Notef(err, "invalid pull / merge request id"))
```

Reply:
Well actually, in the backend, we are speaking about merge/request numbers. Here in the CLI, we are mentioning "id". Should we stick to the "number" denomination, and fix it in the CLI argsusage ?

Reply:
My question is not really a wording question. From what I think, the most important question is: I'm not completely keen on checking here if it's an integer. Why don't we simply trust the backend which forwards the query to GitHub/GitLab to know if it's valid or not?

Reply:
Oh, your comment was on the error message line, I thought you were talking about it. 

Yes, we can rely on the backend for the args coherence

Reply:
@EtienneM Hm, well actually go-scalingo takes a number, and have just been released. To take a string, we should make a 6.6.1. Is it worth? :confused:

Reply:
Groumpf.... OK :(